.travis.yml

```diff
@@ -54,7 +54,7 @@ before_install:
   # Install Java goodies
   - mkdir java-files
   # Install PhyloNet
-  - wget http://bioinfo.cs.rice.edu/sites/g/files/bxs266/f/kcfinder/files/PhyloNet_3.5.7.jar -O java-files/PhyloNet_3.5.7.jar
+  - wget http://bioinfo.cs.rice.edu/sites/g/files/bxs266/f/kcfinder/files/PhyloNet_3.5.7.jar -O java-files/PhyloNet_3.5.7.jar --no-check-certificate
```

Comment:
Travis builds fail, because wget cannot retrieve the specified file since there are some issues with the according web certificate. The new flag --no-check-certificate makes wget ignore the certificate problem and enables the download of the *.jar. That might impose a security problem - but my bet is, that we can ignore it at the moment.